cmd/kured/main.go

```diff
@@ -104,6 +114,8 @@ func main() {
 		"when seconds is greater than zero, skip waiting for the pods whose deletion timestamp is older than N seconds while draining a node (default: 0)")
 	rootCmd.PersistentFlags().DurationVar(&drainTimeout, "drain-timeout", 0,
 		"timeout after which the drain is aborted (default: 0, infinite time)")
+	rootCmd.PersistentFlags().DurationVar(&rebootDelay, "reboot-delay", 0,
+		"delay reboot for this duration (default: 0, disabled)")
```

Comment:
"delay reboot for this duration after drain and exclusion from any ELB"? I feel this will be more understandable by someone not reading the code.